device/ipv4.yaml

```diff
@@ -57,7 +59,23 @@ components:
           type: string
           format: ipv4
           default: 0.0.0.0
-        
+
+    Device.Ipv4ManualGatewayMAC:
+      description: >- 
```

Comment:
Instead of adding a new object , would it make sense to directly add the fields to the IPv4/IPv6 objects ( I think we just have IP/Mask/Gateway and these are 2 more ) . Something like : 
ip1.enable_configured_gw_mac  = true
 ip1.configured_gw_mac  = "aa:aa:aa:aa:aa:aa"
 vs current
ip1.manual_gateway_mac.enable = true
ip1.manual_gateway_mac.address = "aa:aa:aa:aa:aa:aa"
@ajbalogh also for comments. Not sure myself . Just a thought.

Reply:
I like it as an optional object as it encapsulates multiple properties, comments and functionality.

Reply:
Ok , lets keep embedded object then.